server/project/api/events_handler.py

```diff
@@ -9,6 +9,17 @@
 from project.error_handlers import *
 import datetime as dt
 
+
+def verify_at_least_1_day_available(availability: Availability) -> int:
+    """Counts the number of days selected as available and returns the number
```

Comment:
Will this function break if availability is `None` ?

Reply:
`if 'availability' in data and not verify_at_least_1_day_available(data['availability']):
    raise NoDayAvailable`